app/presenters/content_item/organisation_branding.rb

```diff
@@ -38,13 +38,13 @@ def default_organisation
     # Remove this in the future after migrating organisations to the content store API,
     # and updating them with the correct brand in the actual store.
     def organisation_brand(organisation)
-      brand = organisation["details"]["brand"]
-      brand = "executive-office" if executive_order_crest?(organisation)
+      brand = organisation["details"]["brand"] if organisation
+      brand = "executive-office" if executive_order_crest?(organisation) if organisation
```

Comment:
Chaining `if` conditionals like this isn't valid syntax, it looks different from the change in https://github.com/alphagov/government-frontend/pull/726/commits/3015f55bde04e5c76d97bc6dff996e92ec90d4cd, I think adding:
```ruby
return unless organisation
```
might be cleaner than multiple if conditions throughout the method.